pkg/operator/ceph/object/s3_operations.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Rook Authors. All rights reserved.
+Copyright 2020 The Rook Authors. All rights reserved.
```

Comment:
@leseb we have another [bucket/s3-handlers.go](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/bucket/s3-handlers.go) which do have apis for creating s3-client and buckets plus apis related to bucket policy, do like to merge both or atleast remove redundant  functions

Reply:
I think I've started with that but it wasn't working quite well, I can look again.

Reply:
Refactored! Thanks for pointing that out!